packages/ERTP/src/index.js

```diff
@@ -5,12 +5,5 @@ export * from './amountMath.js';
 export * from './issuerKit.js';
 export * from './typeGuards.js';
 
-/**
- * Importing Baggage from `@agoric/ertp` is deprecated. Import Baggage from
- * `@agoric/vat-data` instead
- *
- * @import {Baggage} from '@agoric/vat-data'
```

Comment:
The only questionable one. See the discussion under "Upgrade Considerations" in the PR comment.